site2/website/pages/en/contact.js

```diff
@@ -80,6 +80,11 @@ class Contact extends React.Component {
               </tbody>
             </table>
 
+            <h2><translate>Stack Overflow</translate></h2>
+            <p><translate>For technical questions, we ask that you post them to</translate>
+              <a href="https://stackoverflow.com/" target="_blank"> Stack Overflow </a> <translate>using the tag “apache-pulsar”.  
```

Comment:
Perhaps the link could be directly to the apache-pulsar tag. https://stackoverflow.com/tags/apache-pulsar

Reply:
I agree

Reply:
Agreed will make the change...